inc/formanswer.class.php

```diff
@@ -75,9 +76,10 @@ class PluginFormcreatorFormAnswer extends CommonDBTM
 
    public static function getStatuses() {
       return [
-         self::STATUS_WAITING  => __('Waiting', 'formcreator'),
-         self::STATUS_REFUSED  => __('Refused', 'formcreator'),
-         self::STATUS_ACCEPTED => __('Accepted', 'formcreator'),
+         self::WAITING_APPROVAL  => __('Waiting for approval'),
+         self::STATUS_WAITING    => __('Waiting', 'formcreator'),
+         self::STATUS_REFUSED    => __('Refused', 'formcreator'),
+         self::STATUS_ACCEPTED   => __('Accepted', 'formcreator'),
```

Comment:
2 constants shouldn't have the same value.

I misspoke, I was thinking of renaming the 3 constants, as `STATUS_...` is not explicit enough.

```suggestion
         self::APPROVAL_WAITING    => __('Waiting for approval'),
         self::APPROVAL_REFUSED    => __('Refused', 'formcreator'),
         self::APPROVAL_ACCEPTED   => __('Accepted', 'formcreator'),
```